components/butterfly/src/server/inbound.rs

```diff
@@ -109,8 +109,8 @@ impl<'a> Inbound<'a> {
                 }
                 Err(e) => {
                     match e.raw_os_error() {
-                        Some(35) => {
-                            // This is the normal non-blocking result
+                        Some(35) | Some(11) => {
```

Comment:
Good times. We'll certainly have to fix this up - they shouldn't be different between Linux and Mac, given that it's POSIX socket results bubbled up.